web/.editorconfig

```diff
@@ -0,0 +1,15 @@
+root = true
```

Comment:
Per the docs this is not indicating the root of any content, rather:

> root: special property that should be specified at the top of the file outside of any sections. Set to true to stop .editorconfig files search on current file.

Reply:
That line was a bit unclear to me, so I looked around. It seems based on https://github.com/editorconfig/editorconfig/issues/376 that it is in fact referring to a root directory? Since it inherits rules

Reply:
I am a liar and a fool.